us-export-control.md

```diff
@@ -0,0 +1,111 @@
+# US export control process
+
+The United States imposes some controls over the export of cryptography.
+These controls include special allowances for open source software like Node.js.
+
+Best practice based on advice from the Node.js Foundation is to
+follow the "Technology and Software-Unrestricted" (TSU) licensing exception
+procedures. These procedures require the following:
+
+* Sending notification emails listing the locations
+  of our repositories that contain cryptographic software.
+* Validating that our licences are OSI-approved.
+* Ensuring the software is generally available to any
+  member of the public by posting it on a public Internet location.
+* Reviewing our repositories for binaries. If binaries contain
+  cryptography then the corresponding source must be available in the
+  repository as well.
+
+For complete details on the exception refer to
+[CFR-2012-title15-vol2-sec740-13.pdf](https://www.gpo.gov/fdsys/pkg/CFR-2012-title15-vol2/pdf/CFR-2012-title15-vol2-sec740-13.pdf).
+
+## Notification Emails 
+
+Notification emails must be sent listing the locations which contain
+cryptography. Since we regularly create additional copies of the
+node core repository (which contains cryptography) we will list the
+location of the [organization](https://github.com/nodejs)
+as opposed to any specific repository. In addition, we will list
+the website location as it hosts the Node.js binaries.
+
+In order to meet the archival requirements of the (TSU) licencing exception
+(5 years past the last date where the repo contained cryptographic software),
+the foundation will provide an email account that will be
```

Comment:
I think it'd be better if we change,
foundation/Foundation -> Node.js Foundation
at all places. Just to be more clear.

Reply:
Makes sense. I had considered that while writing.  Will update on next edit.